src/common/OptionsMenu/optionsMenu.scss

```diff
@@ -0,0 +1,37 @@
+@import '../../scss/shadows';
+
+.options-menu {
+  overflow-y: auto;
+  color: rgba(72, 63, 86, 0.87);
+  background-color: #fff;
+  border: 1px solid rgba(0, 0, 0, 0.2);
```

Comment:
Please use scss variable `$primaryBorder` instead of `1px solid rgba(0, 0, 0, 0.2)`